### DISCUSSION TITLE
Donation option

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#210] [Ideas]
I do events for a number of charities, and one thought would be to allow a 'donation' to made alongside and ticket purchases. We've often done a 'pay it forward', and/or make a gift to the charity. I'd wondered about the 'ticket donation / pay what you like' option, but of course it is a ticket, which can be confusing for some people.  

In a way I'm thinking the same functionality for creating, operating and paying that kind of thing, but it doesn't get a ticket pdf, or ticket count, etc.. I've used some systems where donation is an additional option with an open type field, that just adds a donation figure and text onto the cart for checkout. 

Would value any thoughts and feedback.

**Comment by daveearley** [comment]
Good suggestion! This will be possible once "products" are added. Products will effectively be tickets, but without an attendee. So you will be able to add a "donation" product.

I'm working on this currently, and hope it have it finished relatively soon.

**Reply by justynpride** [reply]
That sounds amazing. I can see how the products feature could be utilised to solve so many different needs. Excited!
